extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -311,19 +318,44 @@ public function getVotes() {
 		$pageIds = array_keys( $this->getPages() );
 
 		$dbh = wfGetDB( DB_SLAVE );
-		( new \WikiaSQL() )
-			->SELECT_ALL()
+		( new \WikiaSQL() )->SELECT_ALL()
 			->FROM( self::TABLE_VOTE )
-			->WHERE( 'article_id' )->IN( $pageIds )
+			->WHERE( 'article_id' )
+			->IN( $pageIds )
 			->runLoop( $dbh, function ( &$pages, $row ) {
 
 				$this->addVote( [
 					"page_id" => $row->article_id,
 					"user_identifier" => $row->user_id,
-					"timestamp" => $row->time
+					"timestamp" => $row->time,
 				] );
 			} );
 
 		return $this->votes;
 	}
+
+	public function getFollows() {
+		$this->threadIds = [];
+
+		foreach ($this->getPages() as $page) {
+			if ($page->namespace === NS_WIKIA_FORUM_BOARD_THREAD) {
+				$threadIds[$page->raw_title] = $page->id;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$threadIds" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)